modules/videoio/src/cap_libv4l.cpp

```diff
@@ -1319,6 +1320,7 @@ static IplImage* icvRetrieveFrameCAM_V4L( CvCaptureCAM_V4L* capture, int) {
     switch(capture->imageProperties.palette) {
       case VIDEO_PALETTE_RGB24:
       case VIDEO_PALETTE_YUV420:
+      case VIDEO_PALETTE_YUYV:
```

Comment:
Yes. "Some V4L drivers respond to one, some to the other." ( http://linuxtv.org/downloads/v4l-dvb-apis/compat.html#v4l-image-properties )
Somebody use this without v4l2? I think it is not compilable without v4l2. (https://github.com/soyersoyer/opencv/blob/yuyv_pixfmt/modules/videoio/src/cap_libv4l.cpp#L1317)